storybook/constants/siteConstants.js

```diff
@@ -3,7 +3,7 @@ const PATTERNFLY_ORG = 'http://www.patternfly.org/';
 export const BASE_URL = {
   PATTERNFLY_ORG,
   PATTERNFLY_LIBRARY: `${PATTERNFLY_ORG}pattern-library/`,
-  REACT_BOOTSTRAP: 'https://react-bootstrap.github.io/'
+  REACT_BOOTSTRAP: 'https://5c507d49471426000887a6a7--react-bootstrap.netlify.com/'
```

Comment:
Yuck!

Reply:
Yeah, I was disappointed to find they didn't have a cleaner URL for their legacy docs. This is what I found in their readme.